src/main/steps/tasklistresponse/proceedings/ProceedingDetailsMapper.ts

```diff
@@ -96,31 +96,30 @@ export const setProceedingDetails = (UserCase, respondent: Respondent, req: AppR
         };
         proceedingDetails.push(proceedingData);
       }
-      currentOrPreviousProceedings.proceedingsList = proceedingDetails;
     });
   }
-  currentOrPreviousProceedings.haveChildrenBeenInvolvedInCourtCase = req.session.userCase.proceedingsStart;
-  currentOrPreviousProceedings.courtOrderMadeForProtection = req.session.userCase.proceedingsStartOrder;
-  respondentDetails.value.response.currentOrPreviousProceedings = currentOrPreviousProceedings;
-
-  return respondentDetails;
+  Object.assign(currentOrPreviousProceedings, {
+    proceedingsList: proceedingDetails,
+    haveChildrenBeenInvolvedInCourtCase: proceedingsStart,
+    courtOrderMadeForProtection: proceedingsStartOrder,
+  });
+  return currentOrPreviousProceedings;
 };
 
 // eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types
-export const getProceedingDetails = (respondent: Respondent, req: AppRequest): Partial<CaseWithId> => {
-  const respondentDetails = respondent;
-  const currentProceedings = respondentDetails.value.response.currentOrPreviousProceedings;
+export const getProceedingDetails = (respondent: Respondent): Partial<CaseWithId> => {
```

Comment:
Change the method name to mapProceedingDetails

Reply:
Done Vivek